### PR TITLE
Cekees/merge ellisonbg

### PR DESCRIPTION
**Pull request by cekees**
@ahmadia your new roams stack breaks the analysis tool on my machine.  I think it's just missing stuff in the erdc fork that was never merged back. I think this merge should fix it.

### CODE REVIEW

**Comment by cekees** [comment]
whoopsie @ellisonbg , I meant this as a PR  against the ERDC fork.  You're welcome to merge this stuff in, but I was  just planning to test it in our fork  for  now.

**Comment by ellisonbg** [comment]
No problem. I have added you and Aron to the repo so you can review/merge
PRs here.

On Tue, Mar 31, 2015 at 5:01 PM, Chris Kees <notifications@github.com>
wrote:

> whoopsie @ellisonbg <https://github.com/ellisonbg> , I meant this as a PR
> against the ERDC fork. You're welcome to merge this stuff in, but I was
> just planning to test it in our fork for now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ellisonbg/leafletwidget/pull/8#issuecomment-88289068>.
>



-- 
Brian E. Granger
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by cekees** [comment]
Thanks! OK, @ahmadia I verified  that I  can run both our applications with this merged source. I need to take a break, but we may want  to look at the other PR before merging this. @kajiglet added scaling and centering support that seems similar to

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-leaftletwidget
+leafletwidget
```
Comment (ahmadia) [inline README.md]: This should go in regardless.

File: leafletwidget/leaflet.py [hunk 2]
```diff
@@ -411,3 +412,56 @@ def __add__(self, item):
     def _handle_leaflet_event(self, _, content):
         pass
 
+    def fitBounds(self, southwest, northeast):
```
Comment (ahmadia) [inline leafletwidget/leaflet.py]: This all looks good.

**Comment by ahmadia** [comment]
@kajiglet - it appears from the commit summaries and the similar contents on the two PRs that you incorporated some ideas from @sgillies in https://github.com/ellisonbg/leafletwidget/pull/2.

It also looks like you added several robustness improvements to https://github.com/ellisonbg/leafletwidget/pull/2, as well as the ability to work without knowing the current map bounds.  

If I understand all this correctly, and especially if this supercedes https://github.com/ellisonbg/leafletwidget/pull/2, I'd suggest we merge this in.

**Pull request closed without merging**